fluent-react/src/localized.ts

```diff
@@ -45,7 +46,40 @@ export function Localized(props: LocalizedProps): ReactElement {
     );
   }
 
-  return l10n.getElement(children, id, { attrs, vars, elems });
+  let componentToRender: ReactNode | null;
+
+  // Validate that the child element isn't an array that contains multiple
+  // elements.
+  if (Array.isArray(children)) {
+    if (children.length > 1) {
+      throw new Error(
+        "Expected to receive a single React element to localize."
+      );
+    }
+
+    // If it's an array with zero or one element, we can directly get the first
+    // one.
+    componentToRender = children[0];
+  } else {
+    componentToRender = children ?? null;
+  }
```

Comment:
I'm pretty sure that this does effectively the same thing:

```suggestion
  const componentToRender = Children.only(children);
```

Reply:
Almost, though it's definitely nicer to delegate the error handling to React. But `Children.only` also throws when given no children (which Fluent React accepts - it just renders the localised string), when given a text node, or when given an array with a single element. Thus, I did have to still add explicit checks for that: 71dfc622

Reply:
Ah, I was wrong and `Children.only()` is less useful than I remembered. So let's not do that.